dll/win32/comdlg32/filedlg.c

```diff
@@ -1754,6 +1758,26 @@ static LRESULT FILEDLG95_InitControls(HWND hwnd)
         }
   }
 
+#ifdef __REACTOS__
+  if (!handledPath && (!fodInfos->initdir || !*fodInfos->initdir))
+  {
+      /* 2.5. Win2000+: Recently used defext */
+      if (!handledPath && win2000plus) {
+          fodInfos->initdir = heap_alloc(MAX_PATH * sizeof(WCHAR));
+          fodInfos->initdir[0] = '\0';
+
+          FILEDLG95_MRU_load_ext(fodInfos->initdir, fodInfos->defext);
+
+          if (fodInfos->initdir[0] && PathIsDirectoryW(fodInfos->initdir)){
```

Comment:
Spaces between the braces to respect the ambient code style.

Reply:
OK.